hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
+
+function handleInput() {
+    const val= document.getElementById('entry').value;
+    if (isNaN(val)){
+        document.getElementById('alert').textContent='You must enter a number';
+        document.getElementById('alert').style.color = 'red';
+        return;
+    }
+    else if (val < 0) {
+        document.getElementById('alert').textContent='Your number must be positive';
+        document.getElementById('alert').style.color = 'red';
+        return;
+    }
+    const valstr = val.toString();
+    const revstr = [...valstr].reverse().join('');
+    if (valstr === revstr) {
+        document.getElementById('alert').textContent ='Yes. This is a palindrome!';
+        document.getElementById('alert').style.color = 'green';
+        return;
+    }
+    else if (valstr !== revstr) {
+        document.getElementById('alert').textContent='No. Try again.';
+        document.getElementById('alert').style.color = 'red';
+        return;
+    }
+    return;
+}
```

Comment:
Yours has the same strange issue as mine does where it's saying there's no EOF line here, but I bet if you look at your code, it shows up.